### ISSUE TITLE
Attempt to use TorchIO library for loading and transforming data

### ISSUE THREAD

**Issue by dufaultc** [McMasterAI/Radiology-and-AI#77]
-Need to upload nii.gz files
-Unknown if it integrates with Pytorch lightning
-Has inbuilt augmentations and normalizations we could use

-Have dataloading working by weekend
-Then work on applying their normalization/augmentations and training
